openedx/frontend-app-learning
# feat: legacy course navigation

## Description

Add an option to enable the legacy course navigation where clicking a breadcrumb leads to the course index page highlighting the selected section.

## Testing instructions

1. Set ENABLE_LEGACY_NAV='true' and ENABLE_JUMPNAV='false'
2. Open a course unit
3. Click the breadcrumb navigation
4. See that you are redirected to the course index page and that the item corresponding to the selected breadcrumb is highlighted and scrolled to

![image](https://github.com/openedx/frontend-app-learning/assets/1616648/be7dd1ce-c891-43b2-bdce-b6d929c5146b)
![image](https://github.com/openedx/frontend-app-learning/assets/1616648/c72ce197-667e-4c76-9d30-90ff0aa96ffd)
![image](https://github.com/openedx/frontend-app-learning/assets/1616648/70395cfe-28ee-49f3-8554-9538a670c680)
![image](https://github.com/openedx/frontend-app-learning/assets/1616648/3f35a3a3-770f-422f-95e8-81c166e01969)
